### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by xtrahuman**
# portfolio accessibility and update changes
- Page titles - no issues found
- Image text alternatives - updated issues
- Text headings - h2 added to work-profile
- Color contrast - issues found but color was based on design
- Resize - no issues found
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found
- updated all changes as requested by the reviewer

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @xtrahuman, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @xtrahuman  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
-     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-208629130-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { window.dataLayer.push(arguments); }
+        gtag('js', new Date());
+
+        gtag('config', 'UA-208629130-1');
+    </script>
```
Comment (mke2111) [inline index.html:18]: - [ ] On line 26, change the title page to a more meaningful name such  Okpara's portfolio

File: index.html [hunk 2]
```diff
@@ -134,7 +134,7 @@ <h2>Software Developer</h2>
             </div>
```
Comment (mke2111) [inline index.html:134]: - [ ] For all the images for the social media icons, add the necessary alt text giving the description for the respective image.

File: index.html [hunk 3]
```diff
@@ -274,10 +274,10 @@ <h3>Multi-Post Stories</h3>
...
-                                        <li><a href=""></a>css</li>
-                                        <li><a href=""></a>html</li>
-                                        <li><a href=""></a>bootstrap</li>
-                                        <li><a href=""></a>Ruby</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>css</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>html</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>bootstrap</li>
+                                        <li><a href="https://github.com/xtrahuman/"></a>Ruby</li>
                                     </ul>
                                 </div>
```
Comment (mke2111) [inline index.html:283]: - [ ] For the contact us section, the section title being 'Contact Us', ensure to main the hierachy in the header tags. Change this to be within `h2` tags.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Tochukwu Okpara, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**